### PR TITLE
Loose ends

### PR DESCRIPTION
**Pull request by mimilettd**
#

Description of Changes Made: I went through our website and tidied up our views.

Anticipated Code Effects: I don't anticipate there to be any.
(Would this merge affect any other piece of our code?)

Reviewer: Anyone who can review it first - @rickyamparo @samuelssnider @JunePaloma. If this can be merged by lunch, that would be great.

### CODE REVIEW

File: app/controllers/main_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class MainController < ApplicationController
   def index
+    @popular_items = OrderItem.top_three_items
```
Comment (samuelssnider) [inline app/controllers/main_controller.rb]: Just like we planned, nice work!

File: app/models/order_item.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
 class OrderItem < ApplicationRecord
 	belongs_to :order
 	belongs_to :item
+
+	def self.top_three_items
```
Comment (samuelssnider) [inline app/models/order_item.rb]: I think 3 is a perfect number for scale, and for not overwhelming our customers!

File: app/models/order_item.rb [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+		item_hash = self.group(:item_id).count
+		popular_items = item_hash.sort_by {|key, value| value}.to_h
```
Comment (samuelssnider) [inline app/models/order_item.rb]: Nice to have this method return the items sorted by popularity, could be useful on another page possibly.

File: app/views/admin/dashboard/index.html.erb [hunk 4]
```diff
@@ -26,18 +25,19 @@
...
                   <% end %>
                 </tr>
             <% end %>
           </tbody>
       </table>
-    </div>
   </center>
 </div>
 
 <div class="container">
```
Comment (samuelssnider) [inline app/views/admin/dashboard/index.html.erb]: Can't see it until we merge, but looks good from here.

File: app/views/dashboard/index.html.erb [hunk 5]
```diff
@@ -1,3 +1,20 @@
-<h2>This is the User Dashboard</h2>
-
-Your email account is <%= @user.email %>
+<div class="container">
+  <div class="card text-center">
+    <div class="card-header">
+      <ul class="nav nav-tabs card-header-tabs">
+        <li class="nav-item">
```
Comment (samuelssnider) [inline app/views/dashboard/index.html.erb]: Nice, more personal, and much better looking.

File: app/models/order_item.rb (hunk 3 above)
Comment (rickyamparo) [reply app/models/order_item.rb]: How are we determining that an item is popular? What I mean is that it seems it's just grabbing the first three items that are in the orderItems table.

**Comment by samuelssnider** [comment]
@JunePaloma @rickyamparo  should take a look; I'm gonna go ahead and merge.

**Pull request merged**